test/notification_test.exs

```diff
@@ -160,7 +160,7 @@ defmodule NotificationTest do
   test "dynamic configuration with named function" do
     {:ok, _} =
       PN.start_link(
-        configure: {NotificationTest, :configure, baz: :frobnicate},
+        configure: {NotificationTest, :configure, [:bar, :baz]},
         foo: :bar
       )
```

Comment:
We need to assert configure was called. So you can do something like this:

    {NotificationTest, :configure, [self()]},

And in configure you do:

    send(parent, :configured)

And at the end of this test you do:

    assert_received :configured